extensions/blocks/podcast-player/style.scss

```diff
@@ -1,11 +1,134 @@
 /**
- * Frontend (site) styles for Podcast Episodes
+ * Podcast Player block shared styles (editor & front-end).
  */
 
+$episode-v-padding: 15px;
+$episode-h-padding: 10px;
+$episode-status-icon-size: 22px;
+$text-color: #92959a;
+$text-color-hover: #000;
+$text-color-active: $text-color-hover;
+$text-color-error: #db5051;
+$block-bg-color: #fff;
+$block-border-color: $text-color;
+$font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
+$font-size: 16px;
+
+/**
+ * Player's (block) parent element
+ */
 .wp-block-jetpack-podcast-player {
-	border: 1px solid;
+	border: 1px solid $text-color;
+	background-color: $block-bg-color;
+
+	// Player's state classes added to this element:
+	// &.is-playing {} // When audio starts playing
+	// &.is-paused {}  // When audio is paused
+	// &.is-error {}   // When playback error occured
+
+	audio {
+		display: none;
+	}
+}
+
+.podcast-player__episodes {
+	list-style-type: none;
+	display: flex;
+	flex-direction: column;
+
+	margin: 0;
+	padding: $episode-v-padding 0;
 }
 
-.wp-block-jetpack-podcast-player audio {
+.podcast-player__episode {
+	margin: 0;
+	color: $text-color;
+
+	&:hover {
+		color: $text-color-hover;
+	}
+
+	/**
+     * When episode "is-active", it means that it's been clicked by a user to
+     * start playback. Combine this class with the Player's state classes (see
+     * above) to apply styling for different scenarios.
+     */
+	&.is-active {
+		color: $text-color-hover;
+		font-weight: bold;
+	}
+}
+
+.podcast-player__episode-link {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-between;
+	padding: $episode-h-padding $episode-v-padding;
+	font-family: $font-family;
+	font-size: $font-size;
+
+	text-decoration: none;
+	color: inherit;
+
+	.podcast-player__episode.is-active & {
```

Comment:
Personally, I like nesting (2 levels max) better as it prevents writing the same class multiple times, making it more resilient to any style changes. E.g., I bumped into issues before where a class got renamed but not in all places causing (hidden) regressions. I got used to reading this kind of selectors, but I can see how this can be less readable. Would you agree that we can keep this kind of nesting if it's not more than 2 levels deep?

Reply:
I prefer 2 levels max as well. I also think this is pretty hard to read :)